### ISSUE TITLE
Configure devworkspace namespace with user name and email in gitconfig

### ISSUE THREAD

**Issue by metlos** [eclipse/che#20871]
### Is your enhancement related to a problem? Please describe

Che server supports storing the user's name and email in the gitconfig mounted to the Che7 workspaces. This is currently not possible for devworkspaces because devworkspace operator doesn't support this.

### Describe the solution you'd like

Once support for this lands in DWO, we should decide how to implement this in Che. We can either simply take the user name and email from the database record as we do now for che7 workspaces or we can take advantage of the `che.eclipse.org/workspace-namespace-owner-uid` that is required on che-targetted namespaces and draw the information from there directly.

### Additional context

There is an issue for implementing this on the DWO side: https://github.com/devfile/devworkspace-operator/issues/694

**Comment by ibuziuk** [comment]
> We can either simply take the user name

my understanding that the vision is that database will cease to exist in the future releases, so probably this is not the best option to consider

**Comment by ibuziuk** [comment]
@metlos another reason not to fall back on db - https://github.com/eclipse/che-docs/pull/2159#issuecomment-991071601

**Comment by skabashnyuk** [comment]
Closing in favor of  https://github.com/eclipse/che/issues/20937 and  https://github.com/eclipse/che/issues/20938
